rules-tests/DeadCode/Rector/Property/RemoveUnusedPrivatePropertyRector/Fixture/constructor_with_middle_params.php.inc

```diff
@@ -27,9 +27,9 @@ class DoNotRemoveParameterOnConstructorOnlyInMiddleParams
 
 namespace Rector\Tests\DeadCode\Rector\Property\RemoveUnusedPrivatePropertyRector\Fixture;
 
-class DoNotRemoveParameterOnConstructorOnlyInMiddleParams
+final class ConstructorWithMiddleParams
 {
-    public function __construct(int $contentFinder, \stdClass $stdClass)
+    public function __construct(\stdClass $stdClass)
```

Comment:
the `$contentFinder` need to be kept as it is in the first param, which cause a BC break, that's why there is test for `DoNotRemoveParameterInMiddle` test fixture.

Reply:
I see. Could you extend this PR to keep this behavior? There was a lot of unneded checks that could be removed.

Reply:
I will try, do you want me to create separate PR after it got merged?

Reply:
That would be the best 👍 
I fell asleep yesterday waiting for CI, so I merge now.

Reply:
https://github.com/rectorphp/rector-src/pull/2231